unity/Assets/Scripts/AgentManager.cs

```diff
@@ -1846,14 +1868,21 @@ public class MultiAgentMetadata {
 [MessagePackObject(keyAsPropertyName: true)]
 public class ThirdPartyCameraMetadata {
     public int thirdPartyCameraId;
-    public Vector3 position;
-    public Vector3 rotation;
+    public Vector3 position; //to be deprecated
+    public Vector3 rotation; //to be deprecated
+    public Vector3 worldRelativeThirdPartyCameraPosition;
+    public Vector3 worldRelativeThirdPartyCameraRotation;
     public float fieldOfView;
 
     //note these should only be returned with values
     //if the third party camera is a child of the agent
-    public Vector3 agentPositionRelativeThirdPartyCameraPosition;
-    public Vector3 agentPositionRelativeThirdPartyCameraRotation;
+    public Vector3? agentPositionRelativeThirdPartyCameraPosition;
+    public Vector3? agentPositionRelativeThirdPartyCameraRotation;
```

Comment:
Change variable name to `agentRotationRelativeThirdPartyCameraRotation`

Reply:
updated in ef3bfb20c08ec9a7a991b23d7e9008af3c72d132